_posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs/the-na-bug.Rmd

```diff
@@ -30,7 +30,7 @@ library(xts)
 library(dygraphs)
 library(dplyr)
 library(ggplot2)
-data <- read.csv(file.path(here::here(), "_posts/2022-05-02-the-na-bug", "daily_electrolyte_counts.csv"), stringsAsFactors = FALSE) %>%
+data <- read.csv(file.path(here::here(), "_posts/2022-05-09-ooh-na-na-where-are-my-sodium-labs", "daily_electrolyte_counts.csv"), stringsAsFactors = FALSE) %>%
```

Comment:
When I used the rename function to change the date, it changed it to the title within the Rmd, I can change it back to "the-na-bug" if you'd prefer that!

Reply:
I think using the title makes sense